experimental/packages/api-logs/src/types/LoggerOptions.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Attributes } from '@opentelemetry/api';
+
+export interface LoggerOptions {
+  /**
+   * The schemaUrl of the tracer or instrumentation library
+   * @default ''
+   */
+  schemaUrl?: string;
+
+  /**
+   * The domain for the events created
+   * @default ''
+   */
+  eventDomain?: string;
```

Comment:
It isn't clear reading through the docs what an event domain is or what it should be used for

Reply:
This is being defined in the spec in [this PR](https://github.com/open-telemetry/opentelemetry-specification/pull/2676/files#diff-4591b0e37590a78cc4a09f5a6bd9e007bcb5aa27ed67552864f9d80fc1c0d84cR13-R32). Do you think we should provide more description here?

Reply:
It's ok for now but it will certainly be required before a stable release to have good docs

Reply:
You can add in the docs that the eventDomain allows using same event name across different domains, yet be unrelated events.